lib/src/utils/product.dart

```diff
@@ -1,17 +1,55 @@
 
 class Product {
-  String id;
-  String price;
-  String title;
-  String currencyCode;
+  String id = "";
+  String priceForAndroid ="";
+  String title ="";
+  String currencyCode = "";
+  String subscriptionPeriod = "";
+  double priceForIos =0;
+
+  Product.jsonForIOS(String id,
+      double priceForIos,
+      String title,
+      String currencyCode,
+      String subscriptionPeriod){
+    this.id = id;
+    this.priceForIos = priceForIos;
+    this.title = title;
+    this.currencyCode = currencyCode;
+    this.subscriptionPeriod = subscriptionPeriod;
+  }
+
+  Product.jsonForAndroid(String id,
+  String priceForAndroid,
+  String title,
+  String currencyCode,
+  String subscriptionPeriod){
+    this.id = id;
+    this.priceForAndroid = priceForAndroid;
+    this.title = title;
+    this.currencyCode = currencyCode;
+    this.subscriptionPeriod = subscriptionPeriod;
+  }
+
+  factory Product.fromJsonIOS(dynamic json) {
+    print(json);
```

Comment:
Prefer using `debugPrint()` over `print()`, since `print()` will also log to the console during production.